### PR TITLE
fix: support db prefix in sql

### LINKED ISSUE #977: Support create, drop, desc db.table statement
**Is your feature request related to a problem? Please describe.**

Users have to `USE db` before they perform DDL statement or DML statement.
In this issue, we are going to support execute DDL or DML statement without specifying default database.

1. support create db.table
```
CREATE db1.t1 (col1 int, col2 timestamp);
```

```
CREATE INDEX index1 (...) on db1.t1;
```
2. support cmd like desc, show ....
```sql
DESC db1.t1;
```

3. 
```sql
INSERT INTO db1.t1 values(1, 123456789);
```

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by nautaa**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
fix

* **What is the current behavior?** (You can also link to an open issue here)
close #977 
support db prefix in sql, including the following statement.

- [x]  create and drop table.
- [x] create and drop index.
- [x]  insert into ...
- [x]  desc ...

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: hybridse/src/planv2/ast_node_converter.cc [hunk 1]
```diff
@@ -1851,8 +1851,15 @@ base::Status ConvertInsertStatement(const zetasql::ASTInsertStatement* root, nod
...
-    *output = dynamic_cast<node::InsertStmt*>(node_manager->MakeInsertTableNode(table_name, column_list, rows));
+    std::string db_name = "";
+    std::vector<std::string> names;
+    CHECK_STATUS(AstPathExpressionToStringList(root->GetTargetPathForNonNested().value(), names));
+    table_name = names.back();
+    if (names.size() == 2) {
+        db_name = names[0];
+    }
+    *output =
+        dynamic_cast<node::InsertStmt*>(node_manager->MakeInsertTableNode(db_name, table_name, column_list, rows));
```
Comment (jingchen2222) [inline hybridse/src/planv2/ast_node_converter.cc:1863]: Add unit test in ast_node_converter_test.cc and cases/plan/insert.yaml. Verify the prefix db works

File: src/cmd/sql_cmd.h [hunk 2]
```diff
@@ -500,16 +499,25 @@ void HandleCmd(const hybridse::node::CmdPlanNode* cmd_node) {
...
                 return;
             }
-            // TODO(denglong): Should support table name with database name
-            auto table = cs->GetTableInfo(db, cmd_node->GetArgs()[0]);
+
+            std::string table_name = cmd_node->GetArgs().back();
+            auto table = cs->GetTableInfo(db_name, table_name);
             if (table == nullptr) {
-                std::cerr << "table " << cmd_node->GetArgs()[0] << " does not exist" << std::endl;
+                std::cerr << "table " << table_name << " does not exist" << std::endl;
```
Comment (jingchen2222) [inline src/cmd/sql_cmd.h]: "table ${db_name.table_name} does not exist" might be better

**Comment by jingchen2222** [comment]
Add some units to sql_router_test.cc

**Comment by jingchen2222** [comment]
@wuyou10206  We need to Test db.table /Create Index on db.table/Drop db.table/ Desc db,table for v0.5.0

**Review by jingchen2222 — APPROVED** [review]
@nautaa Thanks for your contribution!
LGTM

**Review by dl239 — APPROVED** [review]

File: src/cmd/sql_cmd.h [hunk 3]
```diff
@@ -1283,11 +1310,6 @@ void HandleSQL(const std::string& sql) {
                 return;
             }
 
-            // TODO(denglong): Should support table name with database name
```
Comment (nautaa) [inline src/cmd/sql_cmd.h:1286]: #1808

**Pull request merged**